Source/RapyutaSimulationPlugins/Private/Robots/Turtlebot3/Turtlebot.cpp

```diff
@@ -4,9 +4,10 @@
 
 void ATurtlebot::ConfigureVehicleMoveComponent()
 {
+
     Super::ConfigureVehicleMoveComponent();
     check(RobotVehicleMoveComponent);
     // [FrameIds] will then be used for odom init in Super::InitializeMoveComponent(),
     // where [RobotVehicleMoveComponent's PawnOwner] has been available thus they must be inited beforehands.
-    RobotVehicleMoveComponent->SetFrameIds(TEXT("odom"), TEXT("base_footprint"));
+    // RobotVehicleMoveComponent->SetFrameIds(TEXT("odom"), TEXT("base_footprint"));

```

Comment:
We should avoid commenting out code in the long run to save extra refactoring I think. I guess since you have moved these values to their initial default in the class, we could remove this method overriding?
I thought `odom, base_footprint` were very custom values (are they or they are hardcoded in ROS ?), thus initialized them here and did not set them as default values in the generic parent class `ARobotVehicle`, since other child robot types might have different frame id values?

Reply:
odom base_footprint is very common frame names and many navigation has this as default value as far as I know.
but sometimes use different valus such as base_link